### ISSUE TITLE
Investigate `exactextract` for gridded aggregations

### ISSUE THREAD

**Issue by samlamont** [RTIInternational/teehr#275]
Python bindings are available for the [exactextract](https://github.com/isciences/exactextract?tab=readme-ov-file) package which summarizes gridded data to polygons based on fractional pixel coverage.

It seems to fairly well supported/maintained and used by others CIROH, so it might make sense to implement in TEEHR for the sake of consistency.

First we should understand the implications on performance, etc. compared to the current approach.

**Comment by samlamont** [comment]
Maybe the fetching methods should be pulled out into their own package?

**Comment by mgdenno** [comment]
Yeah, it would seem to make sense that some of the functionality we have should not be in Teehr.  I think we originally added it because you can't evaluate data you don't have,so we needed a way to fetch it.  Perhaps we should split it off and make it its own package now that we have a better sense of what the core functionality is.  Maybe as part of the upcoming doc updates we should try to document what is and isn't in the core vision.

**Comment by samlamont** [comment]
Closing this for now and will open issues in reference to the above bullet points
